libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -36,13 +33,10 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.store.select(getAllBooks).subscribe(books => {
-      this.books = books;
-    });
-
-    this.store.select(getReadingList).subscribe(readingBooks => {
-      this.readingList = readingBooks;
-    });
+    // Using async pipe operator in html instead of subscribing in ts file
+    // No need to unsubscribe manually since we are using async in html
+    // Added change detection to onpush for faster page load
+    this.books$ = this.store.select(getAllBooks);
```

Comment:
Please merge this initialization with line no. 20 like below:
`books$: Observable<ReadingListBook[]> = this.store.select(getAllBooks);`